Qualitätsanforderungen.md

```diff
@@ -0,0 +1,13 @@
+## Qualitätsanforderungen
+
+* Kommentare und Variablendeklarationen werden in Englisch verfasst.
+* Einheitliche Namenskonvention: der Erste Buchstabe einer Variable wird groß geschrieben. Kommentare werden in der Form `// summary: ... Input: ... Output:...` festgelegt.
+* Funktionen werden mit zwei Tabstops, der Rest des Quellcodes mit einem Tabstop eingerückt.
+* In Funktionen werden Rückgabewerte immer geprüft, bevor ein Wert zurück gegeben wird.
+* Eingaben werden immer überprüft und mögliche Fehler durch Exceptions abgefangen.
+* Der Quellcode folgt dem Prinzip KISS (Keep it stupid simple) und DRY (Don't repeat yourself).
+* Das Umbrechen einer Quellcodezeile ist zu vermeiden.Dadurch wird der Quellcode gut lesbar.
+* Die Implementierung muss jederzeit durch andere Projektmitglieder verstanden werden können.
+* Es finden regelmäßig gemeinsame Codereviws statt um die Qualität des Quellcodes hoch zu halten.
```

Comment:
Was heißt "gemeinsam"? Da würde ich eher festlegen von wie vielen (und welchen) Leuten ein PR reviewt werden muss. Und am besten auch gleich festlegen, dass sämtliche Änderungen immer in einem Branch erfolgen und nur über reviewten PR in master gemerget werden